tests/test_save_tiles.py

```diff
@@ -63,7 +63,7 @@ def test_command_line_save_tiles_single_file(sample_svs, tmp_path):
             "--output-path",
             tmp_path,
             "--verbose",
-            "False",
+            "True",
```

Comment:
Is this right?

Reply:
yes, changed it to improve coverage. Previously, default was True which was covered in another test, so I changed here to True as now False is covered in another test.